gtfs/spec/en/reference.md

```diff
@@ -407,14 +407,14 @@ File: **Optional**
 
 Primary Key (`fare_product_id`, `fare_media_id`)
 
-To describe the different types of tickets or fares that can be purchased by riders.
+To describe the different types of tickets or fares, including those directly purchasable by riders and integral virtual entities like transfer costs.
```

Comment:
```suggestion
Used to describe the range of fares available for purchase by riders or taken into account when computing the total fare for journeys with multiple segments, such as transfer costs.

Reply:
A small suggestion to make the cases where we want to define "virtual" fare products more clear.